internal/tiltfile/tiltfile.go

```diff
@@ -122,10 +133,16 @@ func (tiltfile Tiltfile) GetServiceConfig(serviceName string) (*proto.Service, e
 		return nil, fmt.Errorf("internal error: k8sService.dockerFileTag was not a string in '%v'", service)
 	}
 
+	mounts := make([]*proto.Mount, 0, len(service.dockerImage.mounts))
+	for _, mount := range service.dockerImage.mounts {
+		repo := proto.Repo{RepoType: &proto.Repo_GitRepo{GitRepo: &proto.GitRepo{LocalPath: mount.repo.path}}}
```

Comment:
this is kinda gross, but I couldn't find a better way

Reply:
ya, the go APIs for protobuf oneofs are awkward. (i am not a big fan of the Go API for generated protobuf in general)